src/views/product-downloads-view/boundary/components/desktop-client-callout/helpers.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Given a release architecture string,
+ * Return a more human-readable version that represents the architecture.
+ */
+export function humanArch(arch: string): string {
```

Comment:
Swiped these helpers from the existing dot-io work. A bit duplicative, but I figure the dot-io work will be baleeted relatively soon, so didn't want to intertwine this implementation.